src/views/metricsDashboard.ejs

```diff
@@ -0,0 +1,145 @@
+<% include partials/header %>
```

Comment:
Let this file name be `./src/views/admin/index.ejs`

Reply:
I've done this too
Want to begin working on the responsiveness of the page